README.rst

```diff
@@ -67,30 +67,53 @@ installed and running in a Docker Container.
     $ cp poppy/etc/poppy.conf ~/.poppy/poppy.conf
     $ cp poppy/etc/logging.conf ~/.poppy/logging.conf
 
-3. Find the ``[drivers:storage:cassandradb]`` section in
+3. Add the following in your poppy.conf file under DEFAULT
+
+     [DEFAULT]
+      verbose = True
+      datacenter = DC1
+      base_url=http://0.0.0.0:8888
+      debug = True
+      project_id_in_url = True
+
+4. Add the absolute path logging in your poppy.conf
+
+      log_file = /Users/username/poppy/poppy.log
+      log_config_append = /Users/username/.poppy/logging.conf
+
+5. Find the ``[drivers:storage:cassandradb]`` section in
    ``~/.poppy/poppy.conf`` and modify the URI to point
    to your local casssandra cluster::
 
     [drivers:storage:cassandra]
-    cluster = <docker ip>
+    cluster = dockerhost
```

Comment:
in place of dockerhost, I guess it could be hostname of the system or the IP address